test/e2e/cronjob.go

```diff
@@ -46,9 +46,10 @@ const (
 )
 
 var (
-	CronJobGroupVersionResource      = schema.GroupVersionResource{Group: batchv2alpha1.GroupName, Version: "v2alpha1", Resource: "cronjobs"}
-	ScheduledJobGroupVersionResource = schema.GroupVersionResource{Group: batchv2alpha1.GroupName, Version: "v2alpha1", Resource: "scheduledjobs"}
-	removedScheduledJobsVersion      = utilversion.MustParseSemantic("v1.8.0")
+	CronJobGroupVersionResource                = schema.GroupVersionResource{Group: batchv2alpha1.GroupName, Version: "v2alpha1", Resource: "cronjobs"}
+	ScheduledJobGroupVersionResource           = schema.GroupVersionResource{Group: batchv2alpha1.GroupName, Version: "v2alpha1", Resource: "scheduledjobs"}
+	removedScheduledJobsVersion                = utilversion.MustParseSemantic("v1.8.0")
```

Comment:
Perhaps we can combine both `removedScheduledJobsVersion` and `skippedAdoptionWithoutControllerRefVersion`.